<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TextEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuFont">
     <property name="title">
      <string>Font</string>
     </property>
     <addaction name="actionSelect"/>
     <addaction name="separator"/>
     <addaction name="actionScale"/>
     <addaction name="actionScale2"/>
    </widget>
    <widget class="QMenu" name="menuColors">
     <property name="title">
      <string>Colors</string>
     </property>
     <addaction name="actionFontColor"/>
     <addaction name="actionHighlightColor"/>
     <addaction name="actionBackgroundColor"/>
    </widget>
    <addaction name="menuFont"/>
    <addaction name="menuColors"/>
   </widget>
   <widget class="QMenu" name="menuEncoding">
    <property name="title">
     <string>Encoding</string>
    </property>
    <addaction name="actionWin1251"/>
    <addaction name="actionUTF8"/>
    <addaction name="actionUTF16"/>
    <addaction name="actionUTF32"/>
    <addaction name="actionKOI8U"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuEncoding"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionScale"/>
   <addaction name="actionScale2"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="text">
    <string>Select...</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="text">
    <string>Scale +</string>
   </property>
  </action>
  <action name="actionScale2">
   <property name="text">
    <string>Scale -</string>
   </property>
  </action>
  <action name="actionFontColor">
   <property name="text">
    <string>Font Color</string>
   </property>
  </action>
  <action name="actionHighlightColor">
   <property name="text">
    <string>Highlight Color</string>
   </property>
  </action>
  <action name="actionBackgroundColor">
   <property name="text">
    <string>Background Color</string>
   </property>
  </action>
  <action name="actionWin1251">
   <property name="text">
    <string>Win 1251</string>
   </property>
  </action>
  <action name="actionUTF8">
   <property name="text">
    <string>UTF8</string>
   </property>
  </action>
  <action name="actionUTF16">
   <property name="text">
    <string>UTF16</string>
   </property>
  </action>
  <action name="actionUTF32">
   <property name="text">
    <string>UTF32</string>
   </property>
  </action>
  <action name="actionKOI8U">
   <property name="text">
    <string>KOI8-U</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
